core/os_shared.h

```diff
@@ -314,6 +314,39 @@ char *
 get_application_pid(void);
 char *
 get_application_name(void);
+
+/* DR_API EXPORT BEGIN */
+/**
+ * Encodings of an application's argument.
```

Comment:
nit: "command-line argument" (else may be confusing if seen in isolation in the docs)

Reply:
Done.